data.sql

```diff
@@ -41,4 +41,51 @@ INSERT INTO
         weigth_kg
     )
 VALUES
-    ('Devimon', '2017-05-12', 5, true, 11.0);
\ No newline at end of file
+    ('Devimon', '2017-05-12', 5, true, 11.0);
+
+INSERT INTO
+    animals(
+        name,
+        date_of_birth,
+        escape_attempts,
+        neutered,
+        weigth_kg
+    )
+VALUES
+    ('Charmander', '2020-02-08', 0, false, 11.0);
+
+INSERT INTO
+    animals(
+        name,
+        date_of_birth,
+        escape_attempts,
+        neutered,
+        weigth_kg
+    )
+VALUES
+    ('Plantmon', '2021-11-15', 2, true, 5.7);
+
+INSERT INTO
+    animals(
+        name,
+        date_of_birth,
+        escape_attempts,
+        neutered,
+        weigth_kg
+    )
+VALUES
+    ('Squirtle', '1993-04-02', 3, false, 12.13);
+
+INSERT INTO
+    animals(
+        name,
+        date_of_birth,
+        escape_attempts,
+        neutered,
+        weigth_kg
+    )
+VALUES
+    ('Angemon', '2005-06-12', 1, true, 45.0),
```

Comment:
- [ ] Same thing for **Angemon** Kindly insert the right weight (-45) to follow the project requirement.
```sql
   ('Angemon', '2005-06-12', 1, true, -45.0)
```